mapper/orm-coordination-outbox-polling/pom.xml

```diff
@@ -87,6 +87,32 @@
                     </execution>
                 </executions>
             </plugin>
+
+            <plugin>
+                <groupId>org.moditect</groupId>
+                <artifactId>moditect-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>add-module-infos</id>
+                        <configuration>
+                            <module>
+                                <moduleInfo>
+                                    <exports>
+                                        org.hibernate.search.mapper.orm.coordination.outboxpolling.avro.generated.impl to org.apache.avro;
+                                        org.hibernate.search.mapper.orm.coordination.outboxpolling.logging.impl to org.jboss.logging;
```

Comment:
Are you sure you need to export something to Avro if you already have an `opens` directive? If not, I'd remove this section entirely and rely on the configuration from the parent POM.

If you really need to add an export for Avro, can you please reuse the other exports from the parent POM (`parents/public/pom.xml`?). Let's not diverge unless absolutely necessary.